### PR TITLE
feat(docs): add docs site internally to project

### PR DESCRIPTION
**Pull request by cerebrl**
## Summary

Create an internal Markdown-centric `/doc` route for our initial documentation site. The intention is to create a usable docs site that lives side-by-side with the project that is very easy to manage and deploy.

## Libraries uses

- `mdsvex`: https://mdsvex.pngwn.io/ (via npm)
- `Pico.css`: https://picocss.com/ (via CDN)

Docs URL: https://forgerock-web-login-framework-git-docscreate-markdown-docs-cix.vercel.app/docs. Docs site as both a dark and light mode as well.

## Screenshot (dark mode)

Web Login Framework ForgeRock 2022-10-31 at 8 48 38 AM

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,163 +1 @@
 # Contributing to the ForgeRock Web Login Framework
```
Comment (cerebrl) [inline CONTRIBUTING.md:1]: Moving all of this to the docs site. We can now put something more standard for this doc.

File: package.json [hunk 2]
```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-svelte3": "^4.0.0",
     "jest": "^27.5.1",
     "markdown-toc": "^1.2.0",
+    "mdsvex": "^0.10.6",
```
Comment (cerebrl) [inline package.json:74]: This is like MDX and allows us full Markdown files as route pages, or intermix Svelte with Markdown.

File: themes/default/journey.cjs [hunk 3]
```diff
@@ -88,7 +88,7 @@ module.exports = function (theme) {
       backgroundColor: colorLib(theme('colors.body.dark')).fade(0.5).toString(),
     },
     '.password-icon': {
-      color: theme('colors.primary.dark'),
+      color: theme('colors.secondary.dark'),
```
Comment (cerebrl) [inline themes/default/journey.cjs:91]: I found this bug when testing.

File: svelte.config.js [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
     adapter: adapter(),
   },
   // Consult https://github.com/sveltejs/svelte-preprocess
   // for more information about preprocessors
-  preprocess: preprocess(),
+  preprocess: [
+    preprocess({
+      postcss: true,
+    }),
+    mdsvex({ extensions: ['.md'] }),
```
Comment (cerebrl) [inline svelte.config.js:17]: This allows us to use `.md` within the project, and it will be processed like any other Svelte file.

**Review by ryanbas21 — APPROVED** [review]
Looks good, asked on the side about how this bundling would affect the widget package for my own clarity. It does not, they are separate in terms of how a consumer would install the widget package itself. Thanks.

**Pull request merged**